launch/mouse_with_lidar.launch.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 RT Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from ament_index_python.packages import get_package_share_directory
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.actions import IncludeLaunchDescription
+from launch.actions import OpaqueFunction
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch_ros.actions import LifecycleNode
+
+
+def generate_launch_description():
+    declare_lidar = DeclareLaunchArgument(
+        'lidar', default_value='lds',
+        description='LiDAR: urg, lds,'
```

Comment:
```suggestion
        description='LiDAR: lds only, for now.'
```
URGについては未実装だと思うので、URGを使ったサンプル追加時にここを変更するのがいいと思います。